tests/integration-tests/tests/cloudwatch_logging/cloudwatch_logging_boto3_utils.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_log_groups():
+    """Get list of log groups."""
+    logs_client = boto3.client("logs")
+    log_groups = logs_client.describe_log_groups().get("logGroups")
+    LOGGER.debug("Log groups: {0}\n".format(json.dumps(log_groups, indent=4)))
+    return log_groups
+
+
+def get_log_streams(log_group_name):
+    """Get list of log streams."""
+    logs_client = boto3.client("logs")
+    streams = logs_client.describe_log_streams(logGroupName=log_group_name).get("logStreams")
```

Comment:
+ minor: we can avoid to define the logs_client
+ What about ClientError? See previous comment

Reply:
ClientError is ignored because it's assumed that a check was done as to whether or not the log group exists.